doc/api/next_api_changes/removals/26865-AD.rst

```diff
@@ -0,0 +1,6 @@
+``repeat``
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
These will need to be trimmed to the same length as the title (or the docs build will complain).